docs/guide/tutorials/collations.txt

```diff
@@ -1,3 +1,369 @@
+      alternate: <string>,
+      maxVariable: <string>,
+      backwards: <bool>
+   }
+
+The only required parameter is ``locale``\ , which the server parses as
+an `ICU format locale ID <http://userguide.icu-project.org/locale>`_.
+For example, use ``en_US`` to represent US English and ``fr_CA`` to
+represent Canadian French.
+
+For a complete description of the available parameters, see the
+:manual:`MongoDB manual entry </reference/collation/index.html#collation-document>` .
+
+
+Assign a Default Collation to a Collection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example creates a new collection
+called ``contacts`` on the ``test`` database and assigns a default
+collation with the ``fr_CA`` locale. Specifying a collation when you
+create the collection ensures that all queries that are run against the
+``contacts`` collection use the ``fr_CA`` collation, unless the query
+specifies another collation. Any indexes on the new collection also
+inherit the default collation, unless the creation command specifies
+another collation.
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+
+   // Connection URL
+   const url = 'mongodb://localhost:27017';
+
+   // Create a new MongoClient
+   const client = new MongoClient(url);
+
+   async function main(client) {
+     const db = client.db('test');
+
+     await db.createCollection('contacts', {
+       collation: { locale: 'fr_CA' }
+     });
+
+     console.log('Collection created.');
+   }
+
+   // Function to connect to the server and run your code
+   async function run() {
+     try {
+       // Connect the client to the server
+       await client.connect();
+       console.log('Connected successfully to server');
+
+       await main(client);
+     } finally {
+       // Ensures that the client will close when you finish/error
+       await client.close();
+     }
+   }
+
+   // Runs your code
+   run();
+
+Assign a Default Collation to an Index
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To specify a collation for an index, use the ``collation``
+option when you create the index.
+
+The following example creates an index on the ``name``
+field of the ``contacts`` collection, with the ``unique`` parameter
+enabled and a default collation with ``locale`` set to ``en_US``.
+
+.. code-block:: js
+
+   async function main(client) {
+     const collection = client.db('test').collection('contacts');
+
+     const result = await collection.createIndex(
+       { name: 1 },
+       { unique: 1 },
+       { collation: { locale: 'en_US' } }
+     );
+
+     console.log(result);
+   }
+
+To use this index, make sure your queries also specify the same
+collation. The following query uses the above index:
+
+.. code-block:: js
+
+   async function main(client) {
+     const collection = client.db('test').collection('contacts');
+
+     const docs = await collection
+       .find({ city: 'New York' }, { collation: { locale: 'en_US' } })
+       .project(( _id: 0 ))
```

Comment:
shouldn't this be ` .project({ _id: 0 })`(so curly braces rather than parenthesis